### PR TITLE
Investigation product data migration

### PR DESCRIPTION
**Pull request by KyleMacPherson**
https://trello.com/c/I3imrFNG/1520-data-migration-of-4-pieces-of-case-specific-info-from-product-to-case

## Description
Migrate investigation product data from the products table to the investigation products table and remove references to these removed columns

## Checklist:
- [ ] Have you documented your changes in the pull request description?
- [ ] Does the change present any security considerations?
- [ ] Is any gem functionality overloaded? Eg: Devise controller methods being overloaded.
- [ ] Has acceptance criteria been tested by a peer?
- [ ] Has the CHANGELOG been updated? (If change is worth telling users about.)

### General testing (author)
- [ ] Test without JavaScript
- [ ] Test on small screen

### Accessibility testing (author)
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable CSS - does content make sense and appear in a logical order?

### CODE REVIEW

File: app/models/product.rb [hunk 1]
```diff
@@ -4,19 +4,14 @@ class Product < ApplicationRecord
...
     "genuine" => "genuine",
     "unsure" => "unsure"
   }
 
-  enum affected_units_status: {
-    "exact" => "exact",
-    "approx" => "approx",
-    "unknown" => "unknown",
-    "not_relevant" => "not_relevant"
-  }
```
Comment (stdstaff) [inline app/models/product.rb:18]: We probably need a new model class, `InvestigationProduct` that has these enums, and has the `redacted_export_with` declaration inside, so these fields continue to be exported from their new table.

**Review by stdstaff — APPROVED** [review]

File: lib/tasks/rapex_import.rake [hunk 2]
```diff
@@ -55,7 +55,6 @@ module DataImportHelper
       name: "#{brand(notification)}, #{name}, #{field_from_notification(notification, 'type_numberOfModel')}",
       description: field_from_notification(notification, "description"),
       category: field_from_notification(notification, "category"),
-      batch_number: field_from_notification(notification, "batchNumber_barcode"),
```
Comment (slorek) [inline lib/tasks/rapex_import.rake:58]: I think we should add this to the `create_investigation_product` method in this task.

**Review by slorek — CHANGES_REQUESTED** [review]
I think we need to handle the fields differently in the `AddProductToCase` service and the products in the seeds file.

**Review by nasirkhanbeis — APPROVED** [review]
Had a quick look at the review app, regression tested cases and product creation. looks all good.

**Review by slorek — APPROVED** [review]

**Pull request merged**